doc/api/errors.md

```diff
@@ -679,6 +679,13 @@ APIs _not_ using `AbortSignal`s typically do not raise an error with this code.
 This code does not use the regular `ERR_*` convention Node.js errors use in
 order to be compatible with the web platform's `AbortError`.
 
+<a id="ERR_ACCESS_DENIED"></a>
+
+### `ERR_ACCESS_DENIED`
```

Comment:
I am just wondering that this relatively `generic` error message can confuse user between the errors thrown by the platform due to insufficient permissions set through the policy by the user versus access restrictions to some level resources or operations that is propagated through system call abstractions.

Is it possible to either: 
 - embed the `subject` (who denied the access) in the string (`ERR_NODE_SEC_ACCESS_DENIED`) or
 - reflect the `insufficient permission` (`ERR_ACCESS_DENIED_INSUFFICIENT_PERMISSION`) in the string?

Reply:
Well, when you try to access a resource denied by the system you get an `EACCESS` (linux) and an errno which specifies this wasn't handled by Node.js itself. Example:

```
rafaelgss@rafaelgss-desktop:~/repos/os/node$ node -e 'require("fs").readFileSync("/etc/sudoers")'
node:internal/fs/utils:347
    throw err;
    ^

Error: EACCES: permission denied, open '/etc/sudoers'
    at Object.openSync (node:fs:584:3)
    at Object.readFileSync (node:fs:452:35)
    at [eval]:1:15
    at Script.runInThisContext (node:vm:129:12)
    at Object.runInThisContext (node:vm:307:38)
    at node:internal/process/execution:83:21
    at [eval]-wrapper:6:24
    at runScript (node:internal/process/execution:82:62)
    at evalScript (node:internal/process/execution:104:10)
    at node:internal/main/eval_string:50:3 {
  errno: -13,
  syscall: 'open',
  code: 'EACCES',
  path: '/etc/sudoers'
}

Node.js v19.3.0
```

This is quite different from the error thrown by the permission model. Also, we usually have `ERR_*` for Node.js error codes. So, I think we are good with `ERR_ACCESS_DENIED`.